Mlem/Localizable.xcstrings

```diff
@@ -458,6 +458,9 @@
     },
     "Quote" : {
 
+    },
+    "Really super long text" : {
+
```

Comment:
All of these are from a `#Preview`. From Xcode 16 onwards the compiler excludes `#Preview` strings; until then we could either not include them in the commit or use `Text(verbatim: )` to avoid them being interpreted as localized strings
```suggestion
```